library/Rattletrap/ClassAttributeMap.hs

```diff
@@ -205,7 +201,7 @@ getParentClassByName className parentCacheId xs =
            (map
               (\(_, parentClassId, _, _) -> parentClassId)
               (filter
-                 (\(_, _, cacheId, _) -> cacheId == parentCacheId)
+                 (\(_, _, cacheId, _) -> cacheId <= parentCacheId)
```

Comment:
😆 This worked great for thousands of replays, then all of a sudden it broke. At least I only had to change one character to fix it!